content/blog/1-from-gantt-charts-to-generative-ai-how-agentic-ai-is-revolutionizing-project-management.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Gen AI for PM series- Part 1: From Gantt Charts to Generative AI: How
+  Agentic AI is Revolutionizing Project Management"
+date: 2025-08-27T15:04:35.590Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Agentic AI
+  - Project Manager
+  - Scurm Master
+  - Gen AI for PM
+  - Gantt chart
+---
+![]()
+
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+## **Introduction: From Static Plans to Intelligent Partners**
+
+Picture this: You’re a project manager juggling deadlines, budget limits, and a flood of emails. You’ve got your trusted Gantt chart open, but within hours, it’s already outdated. A supplier delay changes Task A, which cascades into Tasks B and C, and suddenly your neat timeline looks like spaghetti. Sound familiar?
```

Comment:
Suggestion, as you use PM(s) later in the post:

Picture this: You’re a project manager (PM) juggling deadlines,...